packages/sitecore-jss-cli/src/create/index.test.ts

```diff
@@ -131,120 +115,3 @@ describe('applyHostNameToSitecoreConfig', () => {
-                    }${appName}</templates>
-                  </paths>
-                </templates>
-              </content>
-            </schema>
-          </${appName}GraphQLEndpoint>
-        </endpoints>
-      </GraphQL>
-    </api>
-  </sitecore>
-</configuration>
-    `;
-  };
-
-  it('should apply name using defaults', () => {
-    const config = mockConfig('FooApp', '/somewhere/else/');
-    const result = applyNameToSitecoreConfig(config, 'Bar');
-    expect(result).to.match(/<site ((.|\n|\r)*?)name="Bar"/, 'site name');
-    expect(result).to.match(
-      /<site ((.|\n|\r)*?)rootPath="\/sitecore\/content\/Bar"/,
-      'site root path'
-    );
-    expect(result).to.match(/<app ((.|\n|\r)*?)name="Bar"/, 'app name');
-    expect(result).to.match(
-      /<app ((.|\n|\r)*?)sitecorePath="\/sitecore\/content\/Bar"/,
-      'app sitecorePath'
-    );
-    expect(result).to.match(
-      /<app ((.|\n|\r)*?)graphQLEndpoint="\/api\/Bar"/,
-      'app graphQLEndpoint'
-    );
-    expect(result).to.match(
-      /<BarGraphQLEndpoint ((.|\n|\r)*?)url="\/api\/Bar"/,
-      'GraphQL endpoint'
-    );
-  });
-
-  it('should apply name using replaceName', () => {
-    const config = mockConfig('FooApp', '/somewhere/else/');
-    const result = applyNameToSitecoreConfig(config, 'Bar', 'FooApp');
-    expect(result).to.match(/<site ((.|\n|\r)*?)name="Bar"/, 'site name');
-    expect(result).to.match(
-      /<site ((.|\n|\r)*?)rootPath="\/somewhere\/else\/Bar"/,
-      'site root path'
-    );
-    expect(result).to.match(/<app ((.|\n|\r)*?)name="Bar"/, 'app name');
-    expect(result).to.match(
-      /<app ((.|\n|\r)*?)sitecorePath="\/somewhere\/else\/Bar"/,
-      'app sitecorePath'
-    );
-    expect(result).to.match(
-      /<app ((.|\n|\r)*?)graphQLEndpoint="\/somewhere\/else\/Bar"/,
-      'app graphQLEndpoint'
-    );
-    expect(result).to.match(
-      /<BarGraphQLEndpoint ((.|\n|\r)*?)url="\/somewhere\/else\/Bar"/,
-      'GraphQL endpoint'
-    );
-  });
-
-  it('should not apply name using replaceName if no match', () => {
-    const config = mockConfig('FooApp', '/somewhere/else/');
-    const result = applyNameToSitecoreConfig(config, 'Bar', 'BarApp');
-    expect(result).to.not.match(/<site ((.|\n|\r)*?)name="Bar"/, 'site name');
-    expect(result).to.not.match(
-      /<site ((.|\n|\r)*?)rootPath="\/somewhere\/else\/Bar"/,
-      'site root path'
-    );
-    expect(result).to.not.match(/<app ((.|\n|\r)*?)name="Bar"/, 'app name');
-    expect(result).to.not.match(
-      /<app ((.|\n|\r)*?)sitecorePath="\/somewhere\/else\/Bar"/,
-      'app sitecorePath'
-    );
-    expect(result).to.not.match(
-      /<app ((.|\n|\r)*?)graphQLEndpoint="\/somewhere\/else\/Bar"/,
-      'app graphQLEndpoint'
-    );
-    expect(result).to.not.match(
-      /<BarGraphQLEndpoint ((.|\n|\r)*?)url="\/somewhere\/else\/Bar"/,
-      'GraphQL endpoint'
-    );
-  });
-});
```

Comment:
These replaceName tests are still valuable, no? Even though `applyNameToSitecoreConfig` is gone, would still be good to test against Sitecore config xml file using `applyNameReplacement`.

Reply:
Reverted back, good idea